app/src/main/java/com/example/demomvp/data/source/local/preferences/PreferencesHelperImpl.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.demomvp.data.source.local.preferences
+
+import android.content.SharedPreferences
+import com.example.demomvp.utils.PrefsConst.PREF_IS_LOGIN
+
+class PreferencesHelperImpl(sharedPreferences: SharedPreferences) :
+    PreferencesHelper {
+    private var sharedPreferences: SharedPreferences? = null
+
+    init {
+        this.sharedPreferences = sharedPreferences
+    }
```

Comment:
sao phải thêm bước gián tham số như này nữa 
this.sharedPreference với sharedPreferences có gì khác nhau nhỉ?